code/modules/antagonists/fugitive/fugitive_outfits.dm

```diff
@@ -81,3 +83,76 @@
 	W.assignment = "Police Officer"
 	W.registered_name = H.real_name
 	W.update_label()
+
+/datum/outfit/russiancorpse/hunter
+	ears = /obj/item/radio/headset
+	r_hand = /obj/item/gun/ballistic/rifle/boltaction
+
+/datum/outfit/russiancorpse/hunter/pre_equip(mob/living/carbon/human/H)
+	if(prob(50))
+		head = /obj/item/clothing/head/ushanka
+
+/datum/outfit/bountyarmor
+	name = "Bounty Hunter - Armored"
+	uniform = /obj/item/clothing/under/rank/prisoner
+	head = /obj/item/clothing/head/hunter
+	suit = /obj/item/clothing/suit/space/hunter
+	gloves = /obj/item/clothing/gloves/combat
+	shoes = /obj/item/clothing/shoes/jackboots
+	mask = /obj/item/clothing/mask/gas/hunter
+	glasses = /obj/item/clothing/glasses/sunglasses/garb
+	ears = /obj/item/radio/headset
+	l_pocket = /obj/item/tank/internals/plasma/full
+	r_pocket = /obj/item/restraints/handcuffs/cable
+	id = /obj/item/card/id
+	r_hand = /obj/item/flamethrower/full/tank
+
+/datum/outfit/bountyarmor/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(visualsOnly)
+		return
+	var/obj/item/card/id/W = H.wear_id
+	W.assignment = "Bounty Hunter"
+	W.registered_name = H.real_name
+	W.update_label()
+
+/datum/outfit/bountyhook
+	name = "Bounty Hunter - Hook"
+	uniform = /obj/item/clothing/under/rank/prisoner
+	head = /obj/item/clothing/head/scarecrow_hat
+	gloves = /obj/item/clothing/gloves/botanic_leather
+	ears = /obj/item/radio/headset
+	shoes = /obj/item/clothing/shoes/jackboots
+	mask = /obj/item/clothing/mask/scarecrow
+	r_pocket = /obj/item/restraints/handcuffs/cable
+	id = /obj/item/card/id
+	r_hand = /obj/item/gun/ballistic/shotgun/doublebarrel/hook
+
+/datum/outfit/bountygrapple/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(visualsOnly)
+		return
+	var/obj/item/card/id/W = H.wear_id
+	W.assignment = "Bounty Hunter"
+	W.registered_name = H.real_name
+	W.update_label()
+
+/datum/outfit/bountysynth
+	name = "Bounty Hunter - Synth"
+	uniform = /obj/item/clothing/under/rank/prisoner
+	back = /obj/item/storage/backpack
+	suit = /obj/item/clothing/suit/armor/riot
+	shoes = /obj/item/clothing/shoes/jackboots
+	glasses = /obj/item/clothing/glasses/eyepatch
+	r_pocket = /obj/item/restraints/handcuffs/cable
+	ears = /obj/item/radio/headset
+	id = /obj/item/card/id
+	r_hand = /obj/item/extendohand //come on, do something better than this
+
+/datum/outfit/bountysynth/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(visualsOnly)
+		return
+	H.set_species(/datum/species/synth)
+	H.set_species(/datum/species/synth)//setting twice removes the skin
```

Comment:
This sounds off.

Reply:
it's definitely off

Reply:
What's the other way to do this?

Reply:
Fix whatever bug is causing their sprite to not update.

I dont' think working around it is a good idea, just set it once and leave it broken for someone to fix.

Reply:
*Is* this a bug? For some reason I thought that changing species to Synth is intended to preserve your old appearance.

Reply:
You might be right, I do believe that is the lore of synths so it does sound appropriate.

Probably there is a var that can be passed through when changing the species, or a proc that can be called. I'll dig in and see

Reply:
i found a better way to do this through a proc the synth species has but i need to mess around with it a bit more.